optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -879,30 +920,37 @@ Notice how the `@SpringBootTest` annotation's `properties` overwrites the solver
 to terminate as soon as a feasible solution (`0hard/*soft`) is found.
 This avoids hard coding a solver time, because the unit test might run on arbitrary hardware.
 This approach ensures that the test runs long enough to find a feasible solution, even on slow machines.
-But it does not run a millisecond longer than it strictly must, even on fast machines.
+However, it does not run a millisecond longer than it strictly must, even on fast machines.
+
+//How do I run the test?
 
 === Logging
 
-When adding constraints in your `ConstraintProvider`,
-keep an eye on the _score calculation speed_ in the `info` log,
-after solving for the same amount of time, to assess the performance impact:
+After you have completed your {PLANNER} Spring Boot application, you can use logging information to help you fine-tune the constraints in the `ConstraintProvider`. Review the _score calculation speed_ in the `info` log file to assess the impact of changes to your contraints. Run the application in debug mode to show every step that your application takes or use trace logging to every step and every move per stop.
```

Comment:
Typo: contraints.

Following the [one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) rule makes it easier to spot typos like this and doing reviews.